java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestItemUtils.java

```diff
@@ -15,6 +15,7 @@
 import com.microsoft.java.test.plugin.model.TestKind;
 import com.microsoft.java.test.plugin.model.TestLevel;
```

Comment:
TestItemUtils is mixed with JUnit5. could factor for different test platform.

Reply:
Good idea

Reply:
@yaohaizh Updated

Now, the code of parsing `IMethod` into `TestItem` is moved into each TestFramework searchers.